src/components/Form/index.tsx

```diff
@@ -199,114 +257,139 @@ export const Form = ({
-                </MenuItem>
-              ))}
-            </TextField>
-
-            <TextField
-              {...register('statement')}
-              sx={{ ml: 1, mr: 1, width: '22ch' }}
-              margin='dense'
-              variant='outlined'
-              fullWidth
-              label='Statement'
-              key='statement'
-              type='text'
-              multiline={true}
-              maxRows={4}
-            />
-            <TextField
-              {...register('sourceURI')}
-              sx={{ ml: 1, mr: 1, width: '22ch' }}
-              margin='dense'
-              variant='outlined'
-              fullWidth
-              label='Source URI'
-              key='sourceURI'
-              type='text'
-            />
-            <TextField
-              {...register('confidence')}
-              sx={{ ml: 1, mr: 1, width: '22ch' }}
-              margin='dense'
-              variant='outlined'
-              fullWidth
-              label='Confidence'
-              key='confidence'
-              type='number'
-              inputProps={{
-                min: 0.0,
-                max: 1.0,
-                step: 0.1
-              }}
-            />
+              <TextField
+                {...register('confidence')}
+                sx={{ ml: 1, mr: 1, width: '22ch' }}
+                margin='dense'
+                variant='outlined'
+                fullWidth
+                label='Confidence'
+                key='confidence'
+                type='number'
+                inputProps={{
+                  min: 0.0,
+                  max: 1.0,
+                  step: 0.1
+                }}
+              />
+            </Tooltip>
             {!(selectedClaim?.entType === 'CLAIM') && (
               <>
                 {watchClaim === 'rated' ? (
                   <>
-                    <TextField
-                      select
-                      label='Aspect'
-                      {...register('aspect')}
-                      sx={{ ml: 1, mr: 1, width: '22ch' }}
-                      margin='dense'
-                      variant='outlined'
-                      fullWidth
-                    >
-                      {inputOptions.aspect.map((i: string) => (
-                        <MenuItem value={i} key={i}>
-                          {i}
-                        </MenuItem>
-                      ))}
-                    </TextField>
+                    <Tooltip title='A specific dimension being evaluated or rated' placement='right' arrow>
+                      <TextField
+                        select
+                        label='Aspect'
+                        {...register('aspect')}
+                        sx={{ ml: 1, mr: 1, width: '22ch' }}
+                        margin='dense'
+                        variant='outlined'
+                        fullWidth
+                      >
+                        {inputOptions.aspect.map((i: string, claimText: number) => (
```

Comment:
Oh, no! not the second argument. Second argument was ok as `index`. I was talking about the first argument, the `i`. Which needs to be `claimText`.